actions/helm-gke-deploy/action.yaml

```diff
@@ -42,18 +48,19 @@ runs:
     - name: Deploy helm chart
       shell: bash
       run: |
+        command="helm upgrade ${{ inputs.release-name }} ${{ inputs.chart }} --debug --install --wait --timeout ${{ inputs.timeout }}s --force --namespace ${{ inputs.namespace }}"
+
         if $(echo '${{ inputs.values-yaml }}' | jq -e > /dev/null 2>&1); then
           echo "Deploying with multiple value files"
-          helm upgrade --debug --install --wait --timeout 1200s --force \
-            --version ${{ inputs.chart-version }} \
-            $(echo '${{ inputs.values-yaml }}' | jq -r '[ .[] | . = "--values \(.)"] | join(" ")') \
-            --namespace ${{ inputs.namespace }} \
-            ${{ inputs.release-name }} ${{ inputs.chart }}
+          values=$(echo '${{ inputs.values-yaml }}' | jq -r '[ .[] | . = "--values \(.)"] | join(" ")')
+          command="${command} ${values}"
         else
-          echo "Deploying with single value file"
-          helm upgrade --debug --install --wait --timeout 1200s --force \
-            --version ${{ inputs.chart-version }} \
-            --values ${{ inputs.values-yaml }} \
-            --namespace ${{ inputs.namespace }} \
-            ${{ inputs.release-name }} ${{ inputs.chart }}
+          echo "Deploying with single value file"          
+          command="${command} --values ${{ inputs.values-yaml }}"
         fi
+
+        if [ -n "${{ inputs.chart-version }}" ]; then command="${command} --version ${{ inputs.chart-version }}"; fi
+        if [ -n "${{ inputs.post-renderer }}" ]; then command="${command} --post-renderer ${{ inputs.post-renderer }}"; fi
+
+        echo "Deploying with command: ${command}"
+        eval "$command"
```

Comment:
As you know I'm not a bash expert, but I thought using `eval` should be avoided, see for example [Google's shell guide](https://google.github.io/styleguide/shellguide.html#eval). You could instead collect all parameters in an array and then execute `helm "${parameters[@]}"`.

Reply:
I did it without an array now. What do you think?

Reply:
I used an array now